workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -51,6 +57,21 @@ internal List<string> Validate()
                 errors.Add($"Runtime IP \"{RuntimeIp}\" is not a valid IP address.");
             }
 
+            if (string.IsNullOrEmpty(DevAuthTokenDir))
+            {
+                errors.Add($"{GdkToolsConfigurationWindow.DevAuthTokenDirLabel} cannot be empty.");
+            }
+            else if (!DevAuthTokenDir.Equals("Resources") && !DevAuthTokenDir.EndsWith("/Resources"))
+            {
+                errors.Add(
+                    $"{GdkToolsConfigurationWindow.DevAuthTokenDirLabel} must be at root of a Resources folder.");
+            }
+
+            if (DevAuthTokenLifetimeDays < 0 || DevAuthTokenLifetimeDays > 90)
```

Comment:
How can this be out of range if you're using a slider?

Reply:
if you modify the asset directly, it should throw up an error in the UI when loaded

Reply:
I lied - this can't happen as `Validate` is only called from the gdk tools config window